index.html

```diff
@@ -1886,6 +1930,11 @@ <h4>Rate obfuscation</h4>
           observe any abnormal activity such as a high number of pressure state changes
           spanning across multiple states, and set this flag similarly.
         </p>
+        <p>
+          The platforms exposing the [=own contribution estimate=] will fire events more frequently
+          as the estimate may change independent of the global pressure state, but as these are not
+          separate events they are also subject to the rate obfuscation.
```

Comment:
I'm not sure I follow. If we're firing more events with more detail more often, how do the properties of the rate obfuscation remain the same?

Reply:
You do get more events, but the rate can still be obfuscated in the same manner